docs/next/modules/en/pages/use/security/k8s-ingestion-api-keys.adoc

```diff
@@ -1,10 +1,11 @@
 = API Keys
 :description: SUSE Observability
 
-API keys are used for sending telemetry data to SUSE Observability. It now offers two types of API keys:
+API keys are used for sending telemetry data to SUSE Observability. There are three types of API keys:
 
-* Receiver API Key: This key is typically generated during the initial installation of your SUSE Observability instance, and it never expires
-* Ingestion API Key: You can create Ingestion API Keys using the SUSE Observability CLI (STS). These keys offer expiration dates, requiring periodic rotation for continued functionality.
+* **Service Token** - With this key you can scope the data to be ingested.  It also offers expiration dates.  See xref:/use/security/k8s-service-tokens.adoc#_authenticate_using_service_tokens_for_data_ingestion[Service Tokens].
```

Comment:
perhaps an example of the restriction on the data ingestion?

Reply:
good point.  Added it on the service token page though.